frontend/src/addPlace.js

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import ReactDOM from "react-dom";
+
+class AddPlace extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { name: "", x_coord: "", y_coord: "", image: "" };
+  }
+
+  render() {
+    const { name, x_coord, y_coord, image } = this.state;
+    const placeObject = { name, x_coord, y_coord, image };
+
+    return (
+      <div>
+        <h1>Add Place</h1>
+        <textarea
+          placeholder={"Enter a name"}
+          value={this.state.name}
+          onChange={e => {
+            this.setState({ name: e.target.value });
+          }}
+        ></textarea>
```

Comment:
Nit: you should self close tags that don't have children.

Reply:
edited and self-closed!